lia-demo/src/main/AndroidManifest.xml

```diff
@@ -38,6 +38,11 @@
             </intent-filter>
         </activity>
 
+        <activity
+            android:name="com.lithium.community.android.example.LoginActivity"
+            android:label="@string/title_activity_main"
```

Comment:
Same label for both activities?

Reply:
Doesn't matter they are mutually exclusive. Retained LoginActivity just in case we need it for some testing purposes.